liwords-ui/src/gameroom/scorecard.tsx

```diff
@@ -130,11 +130,8 @@ const Turn = (props: turnProps) => {
   );
 };
 
-export const ScoreCard = (props: Props) => {
+export const ScoreCard = React.memo((props: Props) => {
   const el = useRef<HTMLDivElement>(null);
-  useEffect(() => {
-    el.current?.scrollIntoView({ block: 'end', behavior: 'smooth' });
```

Comment:
How do we get the scorecard content to scroll into view (without scrolling the entire window)? is it possible?

Reply:
Ooh! That's why that was there. Yeah, I can do that.